src/profile/Index.vue

```diff
@@ -1,89 +1,171 @@
 <template>
   <BasePage>
     <template v-slot:header>
+      <!-- Todo update other locales -->
       {{ $vuetify.lang.t('$vuetify.profile.index.header', $root.idpConfig.idpName) }}
     </template>
+    
+    <ProfileProgress :profile="{user: $user, alternates, mfa}"/>
 
-    <v-row>
-      <v-col cols="6">
-        <Attribute :name="$vuetify.lang.t('$vuetify.profile.index.username')" :value="$user.idp_username" sameline />
-        <Attribute :name="$vuetify.lang.t('$vuetify.profile.index.lastLogin')" :value="$user.last_login | format" sameline />
-        <Attribute :name="$vuetify.lang.t('$vuetify.profile.index.manager')" :value="$user.manager_email" sameline />
-      </v-col>
-
-      <v-col cols="6">
-        <ProfileProgress :profile="{user: $user, alternates, mfa}"/>
-      </v-col>
-    </v-row>
+    <h3 class="mt-6">{{ $root.idpConfig.appName || 'Idp App'}} {{ $root.idpConfig.idpName }}</h3>
 
-    <v-alert :value="hasUnverifiedEmails" type="error">
-      <span>{{ $vuetify.lang.t('$vuetify.profile.index.unverifiedEmails') }}</span>
-    </v-alert>
+    <h5 class="my-2">{{ $user.isAuthenticated() ? 'Already logged in' : 'Not logged in' }}</h5>
 
     <v-row>
-      <v-col cols="12" sm="6" md="4">
-        <PasswordCard :meta="$user.password_meta"/>
-      </v-col>
+      <v-col>
+        <v-list two-line>
+          <v-card class="mx-auto">
+            <template v-for="(item, index) in coreItems">
+              <v-divider v-if="showDivider(index)" :key="index" inset></v-divider>
 
-      <v-col cols="12" sm="6" md="4">
-        <PasswordRecoveryCard :methods="alternates"/>
-      </v-col>
+              <a class="text-decoration-none" :key="index" :href="item.url" target="_blank">
+                <v-list-item :key="item.title" @click="() => {}">
+                  <v-list-item-avatar>
+                    <v-img :src="item.avatar"></v-img>
+                  </v-list-item-avatar>
+
+                  <v-list-item-content>
+                    <v-list-item-title v-html="item.title"></v-list-item-title>
+                    <v-list-item-subtitle v-html="item.secondary"></v-list-item-subtitle>
+                  </v-list-item-content>
 
-      <v-col cols="12" sm="6" md="4">
-        <DoNotDiscloseCard :dnd="$user.hide"/>
+                  <v-list-item-avatar>
+                    <v-icon>mdi-launch</v-icon>
+                  </v-list-item-avatar>
+                </v-list-item>
+              </a>
+            </template>
+          </v-card>
+        </v-list>
       </v-col>
     </v-row>
 
-    <v-subheader>{{ $vuetify.lang.t('$vuetify.profile.index.2sv') }}</v-subheader>
+    <v-row>
+      <v-col>
+        <h3>Google Workspace</h3>
+      </v-col>
+    </v-row>
 
     <v-row>
-      <v-col cols="12" sm="6" md="4">
-        <TotpCard :meta="mfa.totp"/>
+      <v-col>
+        <h5>Login with your {{org}}'s Google account</h5>
```

Comment:
I suspect we won't want the `'s` after org name (e.g. "Acme Google account", not "Acme's Google account"):
```suggestion
        <h5>Login with your {{org}} Google account</h5>
```